internal/conf/load.go

```diff
@@ -20,6 +20,13 @@ func candidatePaths() []string {
 	}
 }
 
+func normalizeURL(gwURL string) string {
+	if !strings.HasSuffix(gwURL, "/") {
```

Comment:
I think the three lines of code starting here are redundant as TrimSuffix already contains such a check.  Per the docs https://pkg.go.dev/strings#TrimSuffix

> If s doesn't end with suffix, s is returned unchanged.

Reply:
Also relates to the coverage error from CI.  It looks like you added this normalization code but there was already some a few lines later.  So the other code is now uncovered because the URL gets normalized first.  However I do see that the other code would have supported trimming any number of "/" from the end, while I think this new code only supports trimming a single "/" .

I guess that other code would have been better written as TrimRight(gwURL, "/") in the first place...

Reply:
Whoops, I yanked the lines when I meant to cut them. I replaced the HasSuffix and TrimSuffix lines with a single TrimRight line.